fast_id_map has unsound API and unmaintained (#2308)
rustsec/advisory-db: crates/fast_id_map/RUSTSEC-0000-0000.md

```toml
[advisory]
id = "RUSTSEC-0000-0000"
package = "fast_id_map"
date = "2025-05-06"
informational = "unsound"
url = "https://github.com/Bruce0203/fast_map"
categories = ["memory-corruption"]

[affected.functions]
"fast_id_map::FastMap::get" = ["<= 0.1.0"]

[versions]
patched = []
unaffected = []
```

# soundness issue and unmaintained

> `fast_id_map::FastMap::get` lacks of sufficient checks to its parameter index and used in unsafely get element in vector.
> `fast_id_map` is unmaintained.

Reviewer: ```suggestion
`FastMap::get()` lacks sufficient checks to its parameter index and is used to unsafely get a `Vec` element.

`fast_id_map` is unmaintained.
```
